core/tests/protractor_desktop/learnerDashboard.js

```diff
@@ -153,19 +204,239 @@ describe('Learner dashboard functionality', function() {
     await libraryPage.findExploration(EXPLORATION_SINGING);
     await libraryPage.addSelectedExplorationToPlaylist();
     await learnerDashboardPage.get();
-    await learnerDashboardPage.navigateToPlayLaterExplorationSection();
+    await learnerDashboardPage.navigateToCommunityLessonsSection();
     await learnerDashboardPage.expectTitleOfExplorationSummaryTileToMatch(
       EXPLORATION_SINGING);
     await users.logout();
   });
 
+  it('should display correct topics in edit goals, suggested for you section,' +
+    ' current goals and continue where you left off section', async function() {
+    var TOPIC_NAME = 'Topic 1';
+    var TOPIC_URL_FRAGMENT_NAME = 'topic-one';
+    var TOPIC_DESCRIPTION = 'Topic description';
+    await users.createAndLoginCurriculumAdminUser(
+      'creator@learnerDashboard1.com', 'learnerDashboard1');
+    var handle = await browser.getWindowHandle();
+    await learnerDashboardPage.get();
+    await learnerDashboardPage.navigateToHomeSection();
+    await learnerDashboardPage.expectNumberOfTopicsInSuggestedForYou(0);
+    await learnerDashboardPage.navigateToProgressSection();
+    await learnerDashboardPage.expectNumberOfStoriesInCompletedStory(0);
+    await topicsAndSkillsDashboardPage.get();
+    await topicsAndSkillsDashboardPage.createTopic(
+      TOPIC_NAME, TOPIC_URL_FRAGMENT_NAME, TOPIC_DESCRIPTION, false);
+    await topicEditorPage.expectNumberOfStoriesToBe(0);
+    await topicEditorPage.createStory(
+      'Story Title', 'story-one', 'Story description',
+      '../data/test_svg.svg');
+    await storyEditorPage.returnToTopic();
+
+    await topicEditorPage.expectNumberOfStoriesToBe(1);
+    await topicEditorPage.expectStoryPublicationStatusToBe('No', 0);
+    await topicEditorPage.navigateToStoryWithIndex(0);
+    await storyEditorPage.updateMetaTagContent('story meta tag');
+    await storyEditorPage.saveStory('Added meta tag.');
+    await storyEditorPage.publishStory();
+    await storyEditorPage.returnToTopic();
+    await topicEditorPage.expectStoryPublicationStatusToBe('Yes', 0);
+
+    var url = await browser.getCurrentUrl();
+    var topicId = url.split('/')[4].slice(0, -1);
+    await general.closeCurrentTabAndSwitchTo(handle);
+    await adminPage.editConfigProperty(
+      'The details for each classroom page.',
+      'List',
+      async function(elem) {
+        elem = await elem.editItem(0, 'Dictionary');
+        elem = await elem.editEntry(4, 'List');
+        elem = await elem.addItem('Unicode');
+        await elem.setValue(topicId);
+      });
+    await topicsAndSkillsDashboardPage.get();
+    (
+      await
+      topicsAndSkillsDashboardPage.createSkillWithDescriptionAndExplanation(
+        'Skill Learner Dashboard1', 'Concept card explanation', false));
+    await topicsAndSkillsDashboardPage.get();
+    await topicsAndSkillsDashboardPage.navigateToSkillsTab();
+    await topicsAndSkillsDashboardPage.assignSkillToTopic(
+      'Skill Learner Dashboard1', 'Topic 1');
```

Comment:
Can you use the constant in place of `Topic 1`?

Reply:
Done.